.factory/automation.yml

```diff
@@ -65,6 +65,7 @@ build:
         bazel test //pattern/... --test_output=streamed
         bazel test //logic/... --test_output=streamed
         bazel test //reasoner/... --test_output=streamed
+        bazel test //server/... --test_output=streamed
```

Comment:
We had forgotten to run the unit tests for `server`, ie the configuration tests, which we add to CI now.